src/Main.cc

```diff
@@ -124,7 +125,12 @@ static void initPython()
   FOREACH_TOOL(X)
 #undef X
 
-  Py_Initialize();
+    PyConfig config;
+    PyConfig_InitPythonConfig(&config);
+    PyConfig_SetBytesArgv(&config, argc, argv); 
+    config.inspect = inspect;
+    Py_InitializeFromConfig(&config);
+    PyConfig_Clear(&config);
```

Comment:
What does this do that is different from Py_Initialize?

Reply:
it initialize the interpreter with a config (see https://docs.python.org/3/c-api/init_config.html#c.PyConfig.PyConfig_InitPythonConfig) which allow us to set the `inspect` flag to drop into the interpreter after the script execution (instead of exiting) as documented in https://docs.python.org/3/c-api/init_config.html#c.PyConfig.inspect